docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md

```diff
@@ -80,6 +79,10 @@ If all of the above fail, the server fails with an error.
 
 Secrets are passed to the Anchor Platform (and Anchor Reference Server) via environment variables, which can be set either through command line or using a `.env` file. A list of supported environment variables and their descriptions can be found at [`example.env`].
 
+### Supported Assets
+
+The Anchor Platform reads the list of supported assets from a json file whose address is configured in the config file under `app-config.app-assets` and defaults to [`assets-test.json`] ([ref](https://github.com/stellar/java-stellar-anchor-sdk/blob/1f84429f0c5d35cee75445686242643fbd8cffa5/platform/src/main/resources/anchor-config-defaults.yaml#L74)).
```

Comment:
Hi Marcelo, this link points to a specific commit instead of the HEAD of main. But I also think pointing to main would cause the line number being inaccurate. 

I don't have much experience in this. Is this the common practice?

Reply:
Yeah, if we want to point to a specific line, we need to use a permalink as I did here... I'd love to reference `main`, but that would become inaccurate 😕 and harder to maintain 🔨 .

I think this approach is easier to maintain, but I'm open to change if there's something better.